### ISSUE TITLE
Provide proper console auto-completion with `cdt-gdb-vscode`

### ISSUE THREAD

**Issue by planger** [eclipse-cdt-cloud/cdt-cloud-blueprint#36]
### Feature Description:

Currently the auto-completion in the console of a [cdt-gdb-vscode](https://github.com/eclipse-cdt-cloud/cdt-gdb-vscode) debug console is just listing found words:
![image](https://user-images.githubusercontent.com/588090/196523534-6cebfa04-4fda-4f4c-8e3b-c45bfe79fc37.png)

For other language this auto-completion supports variables of the actual scope.

Probably this is a `cdt-gdb-vscode`? @jonahgraham

**Comment by jonahgraham** [comment]
Yes - I will transfer issue there. I think that Ericsson did some work on this, at least in Linux, so that it works well with GDB CLI which has good completion support.
